operators/src/processing/raster_vector_join/aggregator.rs

```diff
@@ -93,6 +101,14 @@ impl TypedAggregator {
             TypedAggregator::MeanNumber(aggregator) => aggregator.is_satisfied(),
         }
     }
+
```

Comment:
why not impl Aggregator for TypedAggregator?

Reply:
I tried it, but then the TypedAggregator would need to be generic and this requires a lot of changes. I don't think it's worth the change but maybe you can try to do it